scripts/draw-opmphm

```diff
@@ -1,14 +0,0 @@
-#!/bin/sh
```

Comment:
Is it still possible to draw it?

Reply:
It seems that dot files do not support hypergraphs.